cli/src/pcluster/templates/login_nodes_stack.py

```diff
@@ -208,9 +207,6 @@ def _add_login_nodes_pool_launch_template(self):
                                 "LaunchingLifecycleHookName": (
                                     f"{self._login_nodes_stack_id}-LoginNodesLaunchingLifecycleHook"
                                 ),
-                                "DisableSudoAccessForDefaultUserConfig": get_cloud_config_for_default_user(
-                                    self._config.disable_sudo_access_default_user
-                                ),
                                 "DisableSudoAccessForDefault": "true"
```

Comment:
Why is this always "true"?

Reply:
This is not always true. This got styled into different lines, its actually 
`"DisableSudoAccessForDefault": "true"  if self._config.disable_sudo_access_default_user  else "false",`